test/test_public_bindings.py

```diff
@@ -295,7 +295,8 @@ def check_one_element(elem, modname, mod, *, is_public, is_all):
                 if elem_module is None:
                     why_not_looks_public = "because it does not have a `__module__` attribute"
                 elem_modname_starts_with_mod = elem_module is not None and \
-                    elem_module.startswith(modname) and '._' not in elem_module
+                    elem_module.startswith(allow_dict["being_migrated"].get(modname, modname)) and \
```

Comment:
please add a small inline comment to explain this change (it might be good to link this PR description too)

Reply:
oh woops -- just saw the comment. SHould I add a PR with the comment?